go.mod

```diff
@@ -26,7 +26,6 @@ require (
 	go.uber.org/multierr v1.1.0 // indirect
 	golang.org/x/net v0.0.0-20181023162649-9b4f9f5ad519
 	golang.org/x/sys v0.0.0-20181026203630-95b1ffbd15a5 // indirect
-	google.golang.org/appengine v1.2.0 // indirect
```

Comment:
How are these changes and the onces on `go.sum` connected to this PR?

Reply:
Basically I just ran `go mod tidy` and committed the result. 🙂